docs/assets/js/main.js

```diff
@@ -0,0 +1,14 @@
+let acc = document.getElementsByClassName("video_accordion");
+var i;
+
+for (i = 0; i < acc.length; i++) {
+    acc[i].addEventListener("click", function () {
+        this.classList.toggle("active");
+        var panel = this.nextElementSibling;
+        if (panel.style.display === "block") {
```

Comment:
this could be reduced with a tenary operator. 
panel.style.display = (panel.style.display === "block") ? "none" : "block"